tests/Twig/ParameterExtensionTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Tests\StefanDoorn\SyliusGtmEnhancedEcommercePlugin\Twig;
+
+use PHPUnit\Framework\TestCase;
+use StefanDoorn\SyliusGtmEnhancedEcommercePlugin\Twig\ParameterExtension;
+use Twig\Environment;
+use Twig\Loader\ArrayLoader;
+
+final class ParameterExtensionTest extends TestCase
+{
+    /**
+     * @param string $template
```

Comment:
This docblock is not really required as the method declaration already shows that